exercises/change/change.spec.js

```diff
@@ -0,0 +1,78 @@
+
```

Comment:
You can check if you're missing some *canonical* test comparing this specs with [this json file](https://github.com/exercism/problem-specifications/blob/master/exercises/change/canonical-data.json).